cookbook/core/control_flow/subworkflows.py

```diff
@@ -1,42 +1,42 @@
 """
-Subworkflows
+SubWorkflows
 ------------
 
-Subworkflows are similar to :ref:`launch plans <Launch plans>`, since they allow users to kick off one workflow from inside another.
+Subworkflows are similar to :ref:`launch plans <Launch plans>` since they allow users to kick off one workflow from within another.
 
 What's the difference?
-Think of launch plans as pass by pointer and subworkflows as pass by value.
+Consider launch plans as pass by pointer and subworkflows as pass by value.
 
 .. note::
 
-    The reason why subworkflows exist is that this is how Flyte handles dynamic workflows.
-    Instead of hiding this functionality, we expose it at the user level. There are pros and cons of
-    using subworkflows as described below.
+    Flyte's handling of dynamic workflows necessitates the use of subworkflows.
+    We provide this capability at the user level rather than hiding it. The pros and cons of
+    using subworkflows are discussed below.
 
-When Should I Use SubWorkflows?
+When should I use SubWorkflows?
```

Comment:
Shouldn't we capitalize the sub-heading?

Reply:
I was uncertain about it too. Are we following this everywhere? Because the above heading 'What's the difference isn't following this.

Reply:
Done.